core/shortcut_items.ts

```diff
@@ -105,7 +108,9 @@ export function registerCopy() {
         !workspace.options.readOnly &&
         !Gesture.inProgress() &&
         selected != null &&
+        isDeletable(selected) &&
```

Comment:
I see what it's doing, but this code feels weird: `isDeletable(selected) && selected.isDeletable()`. Without context I would have no idea what eat part of that is doing. Is there anything you can do with the name to make it clearer? Maybe `implementsDeletable(selected) && selected.isDeletable()`?

Reply:
Done!

Reply:
Now draggable, selectable, and deletable are inconsistent. Can you change all of them to `implementsX`?

If no because of other dependencies, better to have them consistently be `isX`.

Reply:
I think the issue is we already have a bunch of type guards that are `isX`, so unless we want to break all of those we're going to have inconsistency anyway. Is it better to have a group of type guards be different or just the one?

Reply:
Okay, then revert to `isDeletable`.